File: Linguagem C++/Vetores/SomaVetor.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    int n;
    double soma,media;

    cout << "Quantos numeros voce vai digitar? ";
    cin >> n;

    double vet[n];

    // Lendo o vetor
    for (int i=0;i<=n-1;i++){
        cout << "Digite um numero: ";
        cin >> vet[i];
    };

    // Imprimindo o vetor
    cout << "VALORES = ";
    for (int i=0;i<=n-1;i++){
        cout << fixed << setprecision(1);
        cout << vet[i] << "  ";
    };
    cout << endl;

    // Soma e media dos vetores
    soma = 0;
    media = 0;
    for (int i=0;i<=n-1;i++){
        soma = soma + vet[i];
    };

    media = soma / n;
    cout << fixed << setprecision(2);
    cout << "SOMA = " << soma << endl;
    cout << "MEDIA = " << media << endl;


    return 0;
}

File: Linguagem Java/Exercicios/SomaVetor.java
import java.util.Locale;
import java.util.Scanner;

public class SomaVetor {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int n;
		double soma,media;
		
		System.out.print("Quantos numeros voce vai digitar? ");
		n = sc.nextInt();
		
		double vet[] = new double[n];
		
		// Lendo o vetor
		for (int i=0;i<n;i++) {
			System.out.print("Digite um numero: ");
			vet[i] = sc.nextDouble();
		};
		
		System.out.println();
		// Imprimindo o vetor
		System.out.print("VALORES = ");
		for (int i=0;i<n;i++) {
			System.out.print(vet[i]+"  ");			
		};
		
		//Somando e fazendo a média
		soma = 0;
		for (int i=0;i<n;i++) {
			soma = soma + vet[i];			
		};
		media = soma / n;
		System.out.println();
		System.out.println("SOMA = "+String.format("%.2f", soma));
		System.out.println("MEDIA = "+String.format("%.2f", media));
		
		sc.close();

	}

}
